src/zenml/materializers/built_in_materializer.py

```diff
@@ -32,6 +32,7 @@ class BuiltInMaterializer(BaseMaterializer):
         str,
         bytes,
         dict,
+        Dict,
```

Comment:
If this solved the issue for dictionary annotations, does it make sense to include `typing.List` and `typing.Tuple` here as well?

Reply:
Actually it didn't. This should maybe be removed. I am just using `dict` still in my implementation.

Reply:
And maybe add a TODO so we can take a look at that, I think that's quite an important issue as `Dict`, `List` are the "correct" way to annotate your function in python <= 3.9 I believe